src/napari_deeplabcut/_widgets.py

```diff
@@ -111,6 +113,7 @@ def _save_layers_dialog(self, selected=False):
 
 
 def on_close(self, event, widget):
+
```

Comment:
Please remove the unnecessary blank lines after function definition (there are more below too).